sdk/react/constants.js

```diff
@@ -12,7 +12,7 @@ UI_SIZES = {
 const BUTTON_NAMES = { // must match Objective-C code expectations & skin.json.
   PLAY: 'Play',
   PLAY_PAUSE: 'PlayPause',
-  FULLSCREEN: 'Fullscreen',
+  FULLSCREEN: 'fullscreen',
```

Comment:
You can't do like that. It'll broke an iOS skin sdk

Reply:
I got it, yes, but we need align these names in BUTTON_NAMES , actually they should be same as used in skin.json and i don't like change it in skin.json because it used on web also

Reply:
For now (until we align all button names in some future)  i reverted it to "Fullscreen" with some changes for react native to work with that "Fullscreen" labeled button.